indico/migrations/versions/20200728_1842_d89abffb0f63_add_registered_only.py

```diff
@@ -0,0 +1,30 @@
+"""Add registered_only column for showing menu entries only to registered users.
```

Comment:
I'd go for something more concise here, e.g. "Add menu_entries.registered_only column"
and `20200728_1842_d89abffb0f63_add_menu_entries_registered_only.py` for the filename